docs/reference/esql/functions/types/cos.asciidoc

```diff
@@ -4,7 +4,7 @@
 
 [%header.monospaced.styled,format=dsv,separator=|]
 |===
-number | result
+angle | result
```

Comment:
is `angle` a valid type?

Reply:
No, but that's the title row - it's the names of the columns.